app/src/main/java/com/reddit/rickandmortyapp/compose/MainComposeActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.reddit.rickandmortyapp.compose
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.annotation.ColorRes
+import androidx.compose.material.*
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.unit.dp
+
+/**
+ * Main Activity for Compose support
+ * If you'd like to use Compose, please update the Android Manifest to reference this Activity
+ */
+class MainComposeActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            Scaffold(
```

Comment:
Theming is a pretty universally used part of Compose, I think we should include either a MaterialTheme or a customized version (like the project wizard generates)